OpenRA.Mods.Common/Activities/Air/TakeOff.cs

```diff
@@ -44,7 +45,10 @@ public override Activity Tick(Actor self)
 				(hasHost ? self.World.Map.CellContaining(host.CenterPosition) : self.Location);
 
 			if (NextInQueue == null)
-				return new AttackMoveActivity(self, move.MoveTo(destination, 1));
+			{
+				Func<Activity> getInner = () => move.MoveTo(destination, 1);
+				return new AttackMoveActivity(self, getInner);
+			}
```

Comment:
You can just use an anonymous Func instead: `return new AttackMoveActivity(self, () => move.MoveTo(destination, 1));`

Reply:
fixed.